lib/rubocop/cop/rspec/scattered_setup.rb

```diff
@@ -23,25 +23,35 @@ module RSpec
       #   end
       #
       class ScatteredSetup < Cop
-        MSG = 'Do not define multiple hooks in the same example group.'
+        MSG = 'Do not define multiple `%<hook_name>s` hooks in the same '\
+              'example group (also defined on line(s) %<lines>s).'
```

Comment:
Nitpick: Would it be overkill to make a simple pluralisation method? E.g.

`'example group (also defined on %<lines>s).'`
```rb
def lines(numbers)
  if n.size == 1
    "line #{numbers.first}"
  else
    "lines #{numbers.join(', ')}"
  end
end
```

Reply:
Very reasonable. Changes pushed.